### PR TITLE
Document Issues

### PR DESCRIPTION
**Pull request by sjpadgett**
- document date over escaped

Fixes  A forum issue

Looking at other document issues so will leave open for a bit.
#### Changes proposed in this pull request:
Removed extra quote.

### CODE REVIEW

**Comment by stephenwaite** [comment]
nice bug fix!

File: controllers/C_Document.class.php [hunk 1]
```diff
@@ -1014,7 +1014,7 @@ function update_action_process(string $patient_id = null, $document_id)
             }
 
             if (preg_match('/^\d\d\d\d-\d+-\d+$/', $docdate)) {
-                $docdate = "'$docdate'";
+                $docdate = "$docdate";
             } else {
                 $docdate = "NULL";
```
Comment (mdsupport) [inline controllers/C_Document.class.php:1019]: Maybe this is a good time as any to get rid of NULL dates. Specific restrictions from the documentation are:

> The value inserted must not have a zero for the month part, the day part, or both parts. This is enforced beginning with MySQL 8.0.22, regardless of the server SQL mode setting.

Comment (stephenwaite) [reply controllers/C_Document.class.php:1019]: @mdsupport , how would `NULL` dates violate the new enforcement of no zeroes?

Comment (gutiersa) [reply controllers/C_Document.class.php:1019]: How about the creation date of the document? It would be in the metadata, no?

**Comment by arnabnaha** [comment]
Thank you guys...:)

**Pull request merged**